README.md

```diff
@@ -63,30 +85,4 @@ jobs:
           jiraHost: 'coltdorsey.atlassian.net'
```

Comment:
Do you want to keep this as coltdorsey or do you want to change it to something generic?

Reply:
Fixed